docs/src/main/asciidoc/datastore.adoc

```diff
@@ -3,7 +3,7 @@
 == Spring Data Cloud Datastore
 
 http://projects.spring.io/spring-data/[Spring Data] is an abstraction for storing and retrieving POJOs in numerous storage technologies.
-Spring Cloud GCP adds Spring Data support for http://cloud.google.com/datastore/[Google Cloud Datastore].
+Spring Cloud GCP adds Spring Data support for https://cloud.google.com/firestore/[Google Cloud Firestore] in Datastore mode.
```

Comment:
I don't think this is enough and people might miss it.
I think we need a note box right after the title, explaining that the integration is fully compatible with Firestore in Datastore Mode, but not with Firestore in Native Model.